js/main.js

```diff
@@ -1 +1,84 @@
 /* Javascript goes here! */
+console.log('Sanity Check');
+
+var mapURL;
+var toF;
+var thisMap;
+
+$('.weather-app').css('background-color', 'rgba(255,255,255,.5)');
+
+$(document).ready(function(){
+  //Initialized description
+  $('.description').append("<strong>Condition Outside:</strong> ");
+
+  $('.submission').click(function(event){
+    event.preventDefault();
+  });
+
+  $('.submission').on('click', function(){
+    var userInput = $('.weather-city').val();
+    requestWeatherData(userInput);
+    console.log(userInput);
+  });
+});
+
+function requestWeatherData(cityName){
+  //AJAX request based on user input
+  $.ajax({
+    method: 'GET',
+    url: 'https://api.openweathermap.org/data/2.5/weather',
+    dataType:'json' ,
+    data: 'q=' + cityName + '&APPID=461bf158b2034c572293a7dfe13219a2',
+    success: onSuccess,
+    error: onError
+  });
+};
+
+function onSuccess(weatherData){
+  $('.results').css('display', 'block');
+  $('.results-city').empty();
+  $('.temperature').empty();
+  $('.humidity').empty();
+  $('.description').empty();
+  $('.flag-icon').empty();
+  $('.city-error').css('display', 'none');
+
+  var toF = parseInt(((((weatherData.main.temp)-273.15)*1.8)+32));
+  console.dir(toF);
+  $('.results-city').append(weatherData.name + ", " + weatherData.sys.country);
+  $('.temperature').append(toF + '°');
```

Comment:
You'll want to look up the character code for the degree symbol. While it may work in some browsers, it will also break in a lot more.